src/server.ts

```diff
@@ -2,37 +2,142 @@ import express, { Request, Response, Application, NextFunction } from 'express';
 import { config } from './config';
 import logger from './logger';
 import { v4 as uuid } from 'uuid';
+import axios from 'axios';
+import * as _ from 'underscore';
```

Comment:
You should add `underscore` library to package.json

Reply:
Added (forgot to install with --save-dev)